src/cloud/components/organisms/Modal/contents/Workspace/WorkspaceModalForm.tsx

```diff
@@ -299,3 +300,7 @@ const WorkspaceModalForm = ({ workspace }: WorkspaceModalFormProps) => {
 }
 
 export default WorkspaceModalForm
+
+const WorkspaceVisibilityDescription = styled.small`
```

Comment:
Using `small` tag doesn't make sense anymore. Please use `p` or `span`.

Reply:
Changed to paragraph